pom.xml

```diff
@@ -516,7 +516,7 @@
             <dependency>
```

Comment:
It's not only airlift version bump, but also initial support for `ColumnNaming`. Add more descriptive commit message.